crates/testlang/outputs/npm/package/package.json

```diff
@@ -0,0 +1,78 @@
+{
+  "name": "@slang-private/slang-testlang",
+  "version": "0.13.1",
+  "description": "A modular set of compiler APIs empowering the next generation of Testlang code analysis and developer tooling. Written in Rust and distributed in multiple languages.",
+  "homepage": "https://nomicfoundation.github.io/slang/",
+  "license": "MIT",
```

Comment:
I also suggest removing all the additional metadata, as they are only used for public facing artifacts (across languages), and are maintained together: description, URLS, contributors, keywords, etc...

Reply:
Done